packages/manager/apps/public-cloud/src/assets/theme/default/_sidebar.less

```diff
@@ -182,6 +182,16 @@ cloud-sidebar {
     .oui-sidebar-link[aria-expanded='true'] {
       background-color: #fff;
     }
+
+    .oui-status_beta {
+      padding: 0 8px;
+      line-height: 15px;
+      font-size: 12px;
+      font-weight: 600;
+      background-color: #47ff78;
```

Comment:
Both background-color and color must be retrieved from the ODS that will brings theming.

Since we are relying on a override I let them as is.